lumi/lumi_jobscript.sh

```diff
@@ -15,11 +15,11 @@
 CONFIG_NAME=CONFIGNAME.yaml #This file should be located in run-anemoi/lumi
 
 #Should not have to change these
-PROJECT_DIR=/pfs/lustrep4/scratch/$SLURM_JOB_ACCOUNT
-CONTAINER_SCRIPT=/pfs/lustrep4/$(pwd)/run_pytorch.sh
-CONFIG_DIR=/pfs/lustrep4/$(pwd)
+PROJECT_DIR=/scratch/$SLURM_JOB_ACCOUNT
+CONTAINER_SCRIPT=$(pwd)/run_pytorch.sh
+CONFIG_DIR=$(pwd)
 CONTAINER=$PROJECT_DIR/aifs/container/containers/anemoi-training-pytorch-2.2.2-rocm-5.6.1-py-3.11.5.sif
-VENV=/pfs/lustrep4/$(pwd)/.venv
+VENV=$(pwd)/.venv
 export VIRTUAL_ENV=$VENV
```

Comment:
Can you replace `$(pwd)` with `$(pwd -P)` in all three places?

Reply:
Done